docs/plugins/conversion_table.py

```diff
@@ -0,0 +1,496 @@
+from dataclasses import dataclass
+from datetime import date, datetime, time, timedelta
+from decimal import Decimal
+from typing import Any, Literal
+
+from pydantic_core import CoreSchema, core_schema
+
+
+@dataclass
+class Row:
+    field_type: type[Any]
+    input_type: type[Any]
+    mode: Literal['lax', 'strict']
+    input_format: Literal['python', 'JSON', 'python & JSON']
+    condition: str | None = None
+    valid_examples: list[Any] | None = None
+    invalid_examples: list[Any] | None = None
+    core_schema: type[CoreSchema] = None
+
+
+table: list[Row] = [
```

Comment:
@samuelcolvin 
The row list will be distributed as JSON lines or another format? in the wheel.
Does the list affect the behavior of the `strict` argument? Or does the list explain the behavior `pydantic-core`?
Also, Does the user custom type use the `Row` class?

Reply:
if including the JSON in the wheel/sdist would help you, we definitely can.

Yes, the `strict`argument is relevant here, that logic is mostly implemented in `pydantic-core`, some is implemented in types in pydantic.

There's currently no way for user custom types to provide `Row` information.